dockerfiles/Dockerfile.x86_64-master_debug

```diff
@@ -1,10 +1,11 @@
-FROM debian:buster-slim as builder
-RUN mkdir -p /fluent-bit/bin /fluent-bit/etc /fluent-bit/log /tmp/fluent-bit-master/
+FROM busybox:1.34.1 as busybox-include
+FROM debian:bullseye-slim as builder
+RUN mkdir -p /fluent-bit/bin /fluent-bit/etc /fluent-bit/log
 ENV DEBIAN_FRONTEND noninteractive
 
 ADD . /source
 
-RUN echo "deb http://deb.debian.org/debian buster-backports main" > /etc/apt/sources.list.d/backports.list
+# hadolint ignore=DL3008
 RUN apt-get update && \
```

Comment:
can we include bash/sh?

Reply:
Well busybox is included currently. We could install the others as you say but I did not want to change the functionality any more than I had to.

Reply:
In fact it is included.